src/cmakeProject.ts

```diff
@@ -2583,8 +2594,8 @@ export class CMakeProject {
         return expandStrings(this.workspaceContext.config.additionalKits, opts);
     }
 
-    async sendFileTypeTelemetry(textDocument: vscode.TextDocument): Promise<void> {
-        const filePath =  util.platformNormalizePath(textDocument.uri.fsPath);
+    async sendFileTypeTelemetry(textDocument: vscode.Uri): Promise<void> {
```

Comment:
This change seems unnecessary. Please revert.

Reply:
This is related to checking if a renamed file is a "CMakeLists.txt" file. in "onDidRenameFiles" we had access to URI.